packages/forms/src/elements/elements.tsx

```diff
@@ -75,38 +70,39 @@ type ErrorMessageProps = {
 type InputElementHeadingProps = {
 	element?: 'div' | 'legend' | null;
 	label?: string;
-	errorId?: string;
 	required?: boolean;
 	hint?: string;
 	meta?: any;
-	inputName?: string;
+	accessibilityHelper: AccessibilityHelper;
 };
 export const InputElementHeading: React.FC<InputElementHeadingProps> = ({
 	element = 'div',
 	label,
-	errorId,
 	required,
 	hint,
 	meta,
-	inputName,
+	accessibilityHelper,
 }) => {
 	return (
 		<>
 			{label && (
-				<FormLabelText
-					element={element}
-					id={inputName ? `${toKebabCase(inputName + 'Label')}` : null}
```

Comment:
Deleted some functionality here (the kebab case naming)?  This was added to aid screen readers on tabular data (so a field displayed in a table could have its column and row context read out too - See Part 3 valuation life expectancy)

Reply:
Ok. Have now applied in the constructor of AccessibilityHelper.